test/extensions/access_loggers/grpc/grpc_access_log_impl_test.cc

```diff
@@ -198,6 +200,46 @@ TEST_F(GrpcAccessLoggerImplTest, WatermarksOverrun) {
       TestUtility::findCounter(stats_store_, "access_logs.grpc_access_log.logs_dropped")->value());
 }
 
+TEST_F(GrpcAccessLoggerImplTest, WatermarksLegacy) {
```

Comment:
Why is this called legacy? Can you add a one liner `//` above this test to explain what it does?